js/bootstrap.js

```diff
@@ -252,7 +252,7 @@
 
         $('#md-content').removeClass ('col-md-12');
         $('#md-content').addClass ('col-md-9');
-        $('#md-content-row').prepend('<div class="col-md-3" id="md-left-column"/>');
+        $('#md-content-row').prepend('<div class="col-md-3"><div id="md-left-column"/></div>');
```

Comment:
Fix for breaking change in jQuery v1.11.1-beta1: it's needed to give the content navigation menu the formerly behavior.